build-tools/packages/build-tools/src/common/tsCompile.ts

```diff
@@ -0,0 +1,122 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import type * as tsTypes from "typescript";
+import path from "path";
+import { getTscUtils } from "./tscUtils.js";
+
+// Any paths given by typescript will be normalized to forward slashes.
+// Local paths should be normalized to make any comparisons.
+const directorySeparator = "/";
+const backslashRegExp = /\\/g;
+function normalizeSlashes(path: string): string {
+	return path.includes("\\") ? path.replace(backslashRegExp, directorySeparator) : path;
```

Comment:
Just FYI - If you prefer a non-regex solution:
```
path.replaceAll("\\","/");
```

Reply:
Of course. I was just grabbed code bit from tsc-mutli without looking at how over complicated it is. It has other uses for those constants, but we don't need that.

Reply:
`string.replaceAll` does not appear to be a thing.
But simplified `path.replace(/\\/g, "/")` should work.